sourcecode/apis/contentauthor/resources/assets/react/components/QuestionContentType/components/Presentation/PresentationLayout.js

```diff
@@ -13,7 +13,7 @@ function PresentationLayout(props) {
     return (
         <div className="selectPresentationContainer">
             <p>
-                <FormattedMessage id="PRESENTATION.CHOOSE_A_PRESENTATION_FORM" />
+                <h2><FormattedMessage id="PRESENTATION.CHOOSE_A_PRESENTATION_FORM" /></h2>
```

Comment:
The surrounding `<p>...</p>` must be removed. Headers are not valid children of paragraphs in HTML.